config/cmake/modules/ResolveCompilerPaths.cmake

```diff
@@ -103,7 +103,6 @@ macro (RESOLVE_LIBRARIES LIBS LINK_LINE)
       endif (_lib)
     endif (token MATCHES "-L([^\" ]+|\"[^\"]+\")")
   endforeach (token)
-  set (_lib "NOTFOUND" CACHE INTERNAL "Scratch variable" FORCE)
```

Comment:
It may be good to remove `_lib` from the cache variables with `unset(_lib CACHE)`.

Reply:
well since lib isn't being set as a cache variable anymore, is that necessary?

`set (_lib "NOTFOUND")`

Reply:
The call to `find_library(_lib ...)` makes `_lib` a cache entry, see e.g. https://cmake.org/cmake/help/v3.12/command/find_library.html.

Reply:
oh good to know. ill add that then.